Library/Homebrew/utils.rb

```diff
@@ -431,6 +431,7 @@ def which_editor
 end
 
 def exec_editor(*args)
+  puts "Editing #{args.join "\n"}"
```

Comment:
I thought it did render more nicely than that case as it's not truncated and the first argument (the typical case) is displayed on the same line.